### ISSUE TITLE
"xFormers not available" error when running the sample code

### ISSUE THREAD

**Issue by 1412-KID** [DepthAnything/Depth-Anything-V2#125]
import cv2
import torch

from depth_anything_v2.dpt import DepthAnythingV2

get error!
/usr/bin/python3 /home/dp/Depth-Anything-V2/test.py 
xFormers not available
xFormers not available

**Comment by 1412-KID** [comment]
![企业微信截图_20240808104033](https://github.com/user-attachments/assets/50a3d7c4-f072-48f5-b863-445c18a97144)

**Comment by 1412-KID** [comment]
![企业微信截图_20240808104033](https://github.com/user-attachments/assets/30b4e851-1361-49db-babc-35e3479e2b32)
I don't know how to solve this problem.

**Comment by LiheYoung** [comment]
Indeed, this warning will not affect the running of the sample code. You can also install the `xformers` library to avoid this warning.

**Comment by Master-cai** [comment]
Hi @LiheYoung !, I'm wondering what's the difference w/wo xformers? Speed or what else?

**Comment by heyoeyo** [comment]
> I'm wondering what's the difference w/wo xformers?

It can help a lot if you run at higher input image sizes, but only works with cuda.
Memory usage drops significantly and if you use float16, there's a significant speed up as well. Here's a [results comparison](https://github.com/heyoeyo/muggled_dpt/tree/main/lib/v1_depthanything/components#xformers) for depth-anything v1 (should be the same for v2).

**Comment by skysurfer123** [comment]
After install a pytorch2.6.0 version that support cuda, my problem solved and I can predict video now!
我也遇到了同样的问题，xformers不安装可以预测图片但是视频好像不行，后来我发现问题是我的pytorch只支持cpu于是重装了一个支持gpu的版本
